Description: <blankon-session>
 Patch for gnome-screenserver in BlankOn.
 .
 gnome-screensaver (3.6.1-7blankon2) uluwatu; urgency=medium
 .
   * Renew patch
Author: AnTo SamalonA <ansamsoftdev@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-11-17

--- gnome-screensaver-3.6.1.orig/src/gnome-screensaver.desktop.in.in
+++ gnome-screensaver-3.6.1/src/gnome-screensaver.desktop.in.in
@@ -5,7 +5,7 @@ _Comment=Launch screensaver and locker p
 Icon=preferences-desktop-screensaver
 Exec=gnome-screensaver
 OnlyShowIn=GNOME;
-AutostartCondition=GNOME3 if-session gnome-fallback
+AutostartCondition=GNOME3 unless-session gnome
 NoDisplay=true
 X-GNOME-Autostart-Phase=Application
 X-GNOME-Autostart-Notify=true
